server/spec/content_versioning_spec.rb

```diff
@@ -43,6 +51,8 @@
   end
 
   it "creates a new content instance when an org updates a shared instance" do
+    pending("candlepin is not running with per-org products") if !@per_org
```

Comment:
I've updated the RSpec version we're using.  If my PR hits before yours, you may need to switch `pending` to `skip`.  RSpec 3 now treats pending tests as failures and skipped tests as warnings.  RSpec 2 treated pending tests as warnings and didn't have skipped tests.